modules/storage/filesystem/btrf_test.go

```diff
@@ -133,145 +134,105 @@ func TestMain(m *testing.M) {
 func basePoolTest(t *testing.T, pool Pool) {
 	t.Run("test mounted", func(t *testing.T) {
 		_, mounted := pool.Mounted()
-		if ok := assert.False(t, mounted); !ok {
-			t.Error()
-		}
+		assert.False(t, mounted)
```

Comment:
is `assert ` alone will set the test to `error`, or u need to change this to `require`. I think `require ` equals calling `Fatal()`

Reply:
assert make the test fails is the predicate is false. So this is fine since this is the last line of the sub-test and that what we want to verify.
I usually use requires only to early exit tests when condition that are needed for the test to be able to run are not met